owenbot.cabal

```diff
@@ -67,6 +67,7 @@ library
     , http-conduit
     , http-types
     , mtl
+    , MonadRandom
```

Comment:
No need to add MonadRandom, the `random` package (already included) has the function `randomRIO :: (a, a) -> m a` which you can use instead of `getRandomR`.